src/pages/Checkout.page.js

```diff
@@ -0,0 +1,53 @@
+const { ShoppingCartPage } = require('./ShoppingCart.page');
+const { generateRandomData } = require('../utils/random_data');
+
+export class CheckoutPage extends ShoppingCartPage {
+    url = '/checkout-step-one.html';
+
+    itemTotalPrice = '[data-test="subtotal-label"]';
+
+    taxPrice = '[data-test="tax-label"]';
+
+    totalPrice = '[data-test="total-label"]';
+
+    get firstName() { return this.page.locator('#first-name'); }
+
+    get lastName() { return this.page.locator('#last-name'); }
+
+    get zipCode() { return this.page.locator('#postal-code'); }
+
+    get continueButton() { return this.page.locator('#continue'); }
+
+    async fillCheckoutForm() {
```

Comment:
```suggestion
    async fillCheckoutForm(firstName, lastName, zipCode) {
``` To make this code more flexible

Reply:
like this?

```
async fillCheckoutForm(firstName, lastName, zipCode) {
        await this.firstName.fill(firstName);
        await this.lastName.fill(lastName);
        await this.zipCode.fill(zipCode);
        await this.continueButton.click();
    }
```

in the test it will look like this
```
        const { firstName, lastName, zipCode } = generateRandomData();

        await checkoutPage.fillCheckoutForm(firstName, lastName, zipCode);
```

Is that right?

Reply:
Yes

Reply:
In test you can see where you get data from